extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoCoffeeHandler.java

```diff
@@ -213,8 +213,6 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                                 e.getMessage());
                         updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
                     }
-                } else if (command.equals(OnOffType.OFF)) {
-                    // do nothing, as WeMo Coffee Maker cannot be switched off remotely
```

Comment:
Hm, at least such a comment seems to my useful.
If you want to remove the empty branch, you could still keep a comment why we don't need to handle the OFF command.

Reply:
Great idea! I'll remove them, but I'll leave the comments that explain why we don't need them. Thanks!